media-sound/guitarix2/guitarix2-0.35.2.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+
+[[ "${PV}" = "9999" ]] && inherit git-r3
+
+PYTHON_COMPAT=( python{2_7,3_{3,4}} )
```

Comment:
Python 3.3 is obsolete (no longer in upstream portage). Can we test this with 3.5 and perhaps 3.6?